### ISSUE TITLE
Reference MQTT mappings

### ISSUE THREAD

**Issue by LP-SwAG** [SoftwareAG/cumulocity-dynamic-mqtt-mapper#13]
Example MQTT topic structure from new IIoT customer:

`<plant>/<line>/<device>_<MeasurementType>
`

This should result in the cy8_Serial: `<plant>_<line>_<device>` and with MeasurementType in the measurement payload

**Comment by switschel** [comment]
@sagIoTPower We should look into how we can add data from the topic to the target data format.
Similar to the topic selector in the first screen for device ID we can re-use that and select a topic as source and map it.

**Comment by sagIoTPower** [comment]
We enhanced the use of topic levels based on your requirements and incorporated this in release 1.1. This was very valuable.
Now you can use any information from the topic levels: 

```
  "_TOPIC_LEVEL_": 
    "plant",
    "line",
    "device"
  ]```

in mappings:
<img width="1728" alt="Screenshot 2022-10-20 at 15 51 21" src="https://user-images.githubusercontent.com/29702059/197259812-eaf79066-6869-4db5-8203-7aa08a6c1a0e.png">
![Use Topic](https://user-images.githubusercontent.com/29702059/197259812-eaf79066-6869-4db5-8203-7aa08a6c1a0e.png)

Could you test if this works for you?!

Please keep in mind that we only address MQTT payloads in JSON format. What format do you see in your use case?

**Comment by switschel** [comment]
This is fixed with the latest release v1.1+
